x/ccv/provider/keeper/punish_validators.go

```diff
@@ -0,0 +1,35 @@
+package keeper
```

Comment:
would it make sense to name the file `punish_validator.go` because it is used to "punish" once validator at a time?

Reply:
Agree.